src/content/docs/highlights/v0.25.md

```diff
@@ -0,0 +1,385 @@
+<https://github.com/ratatui-org/templates>
+
+Currently, we have two templates:
+
+- Simple: A template for bootstrapping a simply `ratatui` application quickly.
+- Async: A more advanced template which uses crates such as `tokio` and `tracing` for creating an
+  async `ratatui` application.
+
+This repository is still under construction so feel free to take a look and help us out.
+Additionally, we really appreciate if you have another template and interested in contributing it.
+
+---
+
+## New/Updated Examples 🆕
+
+We added a new example
+([`ratatui-logo`](https://github.com/ratatui-org/ratatui/blob/main/examples/ratatui-logo.rs)) which
+shows how to render a big text using half blocks.
+
+![ratatui-logo](https://raw.githubusercontent.com/ratatui-org/ratatui/images/examples/ratatui-logo.gif)
+
+We also updated the
+[`colors_rgb`](https://github.com/ratatui-org/ratatui/blob/main/examples/colors_rgb.rs) example to
+add animation and a FPS counter.
+
+Watch the demo [here](https://github.com/ratatui-org/ratatui/blob/images/examples/colors_rgb.mov).
+
+---
+
+## List: Better Construction ✨
+
+`List::new` now accepts `IntoIterator<Item = Into<ListItem>>` which means you can build a `List`
+like following:
+
+```rust
+List::new(["Item 1", "Item 2"])
+```
+
+However, this change will throw a compilation error for `IntoIterator`s with an indeterminate item
+(e.g. empty vectors):
+
+```rust
+let list = List::new(vec![]);
+
+// instead, this should be:
+let list = List::default();
+```
+
+---
+
+## List: Line Alignment 📏
+
+The `List` widget now respects the alignment of `Line`s and renders them as expected.
+
+For example:
+
+```rust
+let items = [
+    Line::from("Left").alignment(Alignment::Left),
+    Line::from("Center").alignment(Alignment::Center),
+    Line::from("Right").alignment(Alignment::Right),
+]
+```
+
+Will result in a list:
+
+```
+Left
+  Center
+     Right
+```
+
+---
+
+## List: `start_corner` -> `direction`
+
+The previous name `start_corner` did not communicate clearly the intent of the method.
+
+A new method `List::direction` and a new enum `ListDirection` were added.
+
+`List::start_corner` is now deprecated.
+
+---
+
+## Layout: Better Construction ✨
+
+We added a convenience function to create a layout with a direction and a list of constraints which
+are the most common parameters that would be generally configured using the builder pattern.
+
+```rust
+let layout = Layout::new(Direction::Horizontal, [
+  Constraint::Percentage(50),
+  Constraint::Percentage(50),
+]);
+```
+
+The constraints can be passed in as any iterator of constraints.
+
+---
+
+## Layout: Allow Configuring Fill 🖌️
+
+The layout split will generally fill the remaining area when `split()` is called.
+
+With this feature, we now allow the caller to configure how any extra space is allocated to the
+`Rect`s. This is useful for cases where the caller wants to have a fixed size for one of the
+`Rect`s, and have the other `Rect`s fill the remaining space.
+
```

Comment:
We should add a code snipped here to demonstrate this. I couldn't pick anything - can someone help me with that?

Reply:
```rust
Layout::new(
    Direction::Horizontal,
    [Constraint::Percentage(50), Constraint::Percentage(25)],
)
.split(frame.size());
```

The default or `SegmentSize::LastTakesRemainder` will extend the last `Rect`.

![image](https://github.com/ratatui-org/website/assets/36198422/ef0f8c1c-c931-4ed9-b5a2-2ce6333285af)

---

```rust
Layout::new(
    Direction::Horizontal,
    [Constraint::Percentage(50), Constraint::Percentage(25)],
)
.segment_size(layout::SegmentSize::None)
.split(frame.size());
```

`SegmentSize::None` will disable.

![image](https://github.com/ratatui-org/website/assets/36198422/90b15c04-8a50-4f6e-98af-58b257575b90)

---

And the last one isn't really clear to me. Anyway I think `None` is the most important.

Reply:
This is great, thanks!